src/openrct2/peep/peep.c

```diff
@@ -11344,6 +11350,12 @@ static void peep_easter_egg_peep_interactions(rct_peep *peep)
  */
 static bool peep_should_watch_ride(rct_map_element *mapElement) {
     rct_ride *ride = get_ride(mapElement->properties.track.ride_index);
+
+    // In case of upcoming references we also have the check in here.
```

Comment:
I find this comment a bit vague. It isn't clear what it's trying to prevent, nor does it say why (I know it's to prevents desyncs, but the comment should really say that).